src/views/CreateOffProduct.vue

```diff
@@ -355,7 +368,7 @@ export default {
     stepItemList() {
       return [
         {
-          title: this.$t('Common.BarcodeType'),
+          title: this.$t('CreateOffProduct.SelectUnknownProduct'),
```

Comment:
In this commit - https://github.com/openfoodfacts/open-prices-frontend/pull/1771/commits/2d843352bb2673797d2d78be53e6d18de998b2de - I renamed "Choose" to "Select", as we use it pretty much everywhere else (1 exception, but thinking of changing as well to simplify)